# HG changeset patch
# User Andreas Weber <octave@josoansi.de>
# Date 1546363755 -3600
#      Tue Jan 01 18:29:15 2019 +0100
# Branch stable
# Node ID 050ec530e6e4b2387a2a7a35fc08765d95f3336e
# Parent  0990e4093c259b3535e4b3c4aa729ef5fb3af280
html_compare_plot_demos.m: Fix texinfo syntax

diff --git a/scripts/testfun/private/html_compare_plot_demos.m b/scripts/testfun/private/html_compare_plot_demos.m
--- a/scripts/testfun/private/html_compare_plot_demos.m
+++ b/scripts/testfun/private/html_compare_plot_demos.m
@@ -41,17 +41,17 @@
 ##
 ## The template parameter refers to a specially formatted html file
 ## which accompanies this m-file script.
 ##
 ## Additional toolkit description can be added to the column header
 ## with a parameter named equal to the toolkit.  For example:
 ##
 ## @smallexample
-## @code{html_compare_plot_demos ({"gnuplot", "fltk"}, "gnuplot", " 4.6 patchlevel 5")}
+## @code{html_compare_plot_demos (@{"gnuplot", "fltk"@}, "gnuplot", " 4.6 patchlevel 5")}
 ## @end smallexample
 ##
 ## @seealso{compare_plot_demos, dump_demos, demo}
 ## @end deftypefn
 
 ## Author: Ben Abbott  <bpabbott@mac.com>
 
 function html_compare_plot_demos (toolkits, varargin)
